src/euphorie/client/browser/webhelpers.py

```diff
@@ -1018,36 +960,51 @@ def can_edit_session(self):
             return False
         return True
 
+    @cached_property
+    def can_inspect_session(self):
+        """Can the user inspect the session?
+
+        If a session is archived or locked, you cannot edit it.
+        This used to mean that you cannot see the answers to risks, or indeed
+        the entire 'identification' phase.  We may want to allow inspecting
+        those parts without being able to edit them.  In the UI we could show
+        the chosen answers for risks, without showing an editable form.
+        """
+        if not self.can_view_session:
+            return False
+        session = self.traversed_session.session
+        if session.is_archived and not self.allow_inspecting_archived_sessions:
+            return False
+        if session.is_locked and not self.allow_inspecting_locked_sessions:
+            return False
+        return True
```

Comment:
I would return a default which is stricter (e.g., `self.is_manager`).

Reply:
I think that would contradict #2672. Non-manager GpB users are supposed to be able to inspect GpB sessions.

Reply:
In fact, I do a simpler version in the related [`daimler.oira` PR](https://github.com/syslabcom/daimler.oira/pull/336/files).

Reply:
I am still concerned by this default.
Like it is now, this needs an override in customers projects to fully have sense.
Anyway it's ok.

Reply:
I reported the default wrong in [my comment above](#r1806178328).  In this Euphorie PR we set `allow_inspecting_locked_sessions = False` (same for archived).

With those defaults, `can_inspect_session` gives the same value as `can_edit_session`.  So with the default settings this PR changes nothing:

* if you can edit a session you can inspect it
* if you can't edit a session you can't inspect it

This PR just makes it possible for policy packages like `daimler.oira` to set their own defaults, and make them dynamic where needed.

That said: I merge.